tests/testthat/test-example.R

```diff
@@ -433,3 +433,11 @@ test_that("can run volatility example", {
   y <- drop(dust_iterate(obj, 0:100))
   expect_lt(diff(range(colMeans(y))), 0.5)
 })
+
+
+test_that("has_openmp can be called statically and normally", {
```

Comment:
Trying to work out whether `libgomp.so` is required - it may not be just using pragma (still unclear to me), but we use some of the runtime commands from the library (https://gcc.gnu.org/onlinedocs/libgomp/Runtime-Library-Routines.html#Runtime-Library-Routines) so I'm fairly sure it's required for us to run, when compiled with openmp

Reply:
ah, good point

Reply:
ok, I've taken a better stab at this in 5c3f354